src/Hosting/DbBackupInterface.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+/*
+ * This file is part of the Acquia Platform: Cloud Data Model.
+ *
+ * (c) Acquia, Inc.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Acquia\Platform\Cloud\Hosting;
+
+/**
+ * A DbBackup corresponds to a backup of a database for a client's application.
+ * It contains:
```

Comment:
@kirbpowell This interface is structured after the internal data model... it should reflect the fields returned by the public API (http://cloudapi.acquia.com/#GET__sites__site_envs__env_dbs__db_backups-instance_route).

Specifically:
* the site ID and db ID fields are not returned by the API.
* the public API does return 'name' and  'link' fields.